documentation/source/user_section/installation/windows.rst

```diff
@@ -4,14 +4,115 @@
+
+1. Installing Prerequisites
+***************************
+
+SCT depends on two pieces of software that must be set up prior to the installation of SCT.
+
+Python 3.7
+^^^^^^^^^^
+
+Since SCT is a Python package, Python must be installed on your system before SCT can be installed.
+
+1. Download a "Windows x86-64" installer from `the Python 3.7.9 download page <https://www.python.org/downloads/release/python-379/>`_.
+
+2. Run the installer file. (**Important:** Before clicking "Install Now", make sure to first check the "Add Python 3.7 to PATH" checkbox at the bottom of the window.)
+
+3. After the installation has finished, open your Start Menu and type Command Prompt, then run it. In the Command Prompt window, type ``python --version`` and press enter.
+
+   (Make sure that you see the text ``Python 3.7.9`` before continuing.)
+
+
+Git for Windows
+^^^^^^^^^^^^^^^
+
+The easiest way to try out different versions of SCT is using Git.
+
+1. Download Git for Windows from `this download page <https://git-scm.com/download/win>`_.
+
+2. Run the installer.
+
+   - You can click "Next" for most of the options in the installer.
+   - However, on the "Adjusting your PATH environment" page, we strongly recommend that you choose the "Use Git and optional Unix tools from the Command Prompt". Selecting this option will provide you with useful Unix utilities such as ``bash``, ``cd``, ``ls``, and more that combine nicely with SCT's command-line tools. In particular, ``bash`` will allow ``sct_run_batch`` to execute bash scripts for batch processing of subjects.
+   - **Note:** If you prefer, you may instead choose to :ref:`install Cygwin<installing_cygwin>` (rather than selecting the "Use Git and optional Unix tools" option) in order to gain access to these same Unix utilities.
+
+3. After the installation has finished, open your Start Menu and type Command Prompt, then run it. In the Command Prompt window, type ``git --version`` and press enter.
+
+   (Make sure that you see the text ``git version <number>`` before continuing.)
+
+
+2. Installing SCT
+*****************
+
+We recommend that you install SCT into a Python virtual environment. To help with this process, SCT provides an installer script that will automate the process of creating a Python virtual environment for you.
+
+1. Download the ``install_sct.bat`` script using `this link <https://downgit.github.io/#/home?url=https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/master/install_sct.bat>`_, then extract the ``install_sct.bat`` file to your User folder (``C:/Users/<username>``).
+
+2. Open your Start Menu and type Command Prompt, then run that program. In the newly-opened Command Prompt window, type ``install_sct.bat`` and press enter.
```

Comment:
NB: This part of the instructions, as currently written, will install SCT from the `master` branch. 

If our plan for releasing SCT on Windows is:

* In `install_sct.bat`, modify the default value of `git_ref` to be `5.6`
* Attach this modified `install_sct_5.6.bat` as an asset to the 5.6 release page.
* Have the user navigate to the release page, download `install_sct-5.6.bat`, then run the script.

Then we'll need to reword these instructions, while also being mindful of how we time this PR with the release itself. (This is because our RTD documentation is set to `latest` and not `stable`, which means that any changes to the documentation are deployed immediately, rather than being tied to the last stable release.)

Reply:
Most of the Windows user are not technical advanced users and are used to simple `.exe` file for every version of the software, so they just download the correct one.

In this context I like the last option `Have the user navigate to the release page, download install_sct-5.6.bat, then run the script.`, i think that it will be easy for them to follow and it will allow them to decide easily which version they want.

Reply:
Ahh, thank you for your comments. They made me realize that I hadn't really tried to run `install_sct.bat` by double-clicking it. This resulted in: 

* https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3762

Reply:
@joshuacwnewton Do you want to wait #3762 issue to be done before merging this PR or do we keep the documentation like that for the moment and we update it in #3762 ?

Reply:
The way our RTD is set up, documentation changes go live immediately, since RTD is set up to track the "latest" (master) rather than the "stable" (current release).

But, I think we want these instructions to only be made available after we've made the release?

So, here's my proposal for the order we should follow:

1. Create the 5.6 release.
2. Merge this PR to master.
3. Make the announcement, and point to the new Windows instructions in our announcement.